minimal_high_density.slim

```diff
@@ -0,0 +1,130 @@
+initialize() {
```

Comment:
I didn't try actually running the model, but I don't see any problems.  :->

Reply:
Runs fine on my end!